hedyorg/hedy
# [TESTING, CYPRESS] Test for creating an account on sign-up page

**Description**

This includes the test for creating an account on the sign-up page.

**Fixes _issue or discussion number_**

Fixes #3731 

**How to test**

Run 
`export ADMIN_USER=admin_user; python app.py` in your virtual environment. This ensures that there is an active admin_user and Hedy is running.

Then you can go to the /tests folder and run: 

`npx cypress run --spec cypress/e2e/signup_create_page`

The spec should have passed!
